lib/screens/backup/backup_destination_screen.dart

```diff
@@ -163,7 +167,9 @@ class BackupDestinationWidget extends StatelessWidget {
       );
     } catch (error) {
       print('Cannot verify backup: $error');
```

Comment:
Remove unused `print`

Reply:
These were already there. I think added by Vasyl. Should I remove all the print statements in the code